symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/properties/JwtCookieProperties.java

```diff
@@ -0,0 +1,17 @@
+package com.symphony.bdk.app.spring.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Configuration Properties for storing Jwt in cookie for the Extension App
+ */
+@Getter
+@Setter
+public class JwtCookieProperties {
+
+  private static final Integer DEFAULT_MAX_AGE = 24*60*60;
+
+  private Boolean enabled = false;
+  private Integer maxAge = DEFAULT_MAX_AGE;
```

Comment:
We could try to use duration directly https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-conversion-duration

Reply:
do you know how can use `@Min` with the `Duration` field?

Reply:
Then maybe a duration cannot be negative, so no need to check `@Min`